app/locales/en/licensesscreen.json

```diff
@@ -1,3 +1,3 @@
 {
-    "license_placeholder": "This app is built under the MIT license."
+    "license_placeholder": "This app is built under the MIT license. It fully respects your privacy and your data remains secure at your device only."
```

Comment:
Again, like I mentioned in the instructions, when you want to change something in the MIT files, please do not edit them 

Instead, copy this new message in `ixnilatisOverride`, both greek and english

Reply:
ok